PR: https://github.com/odoo/odoo/pull/

From: 7978d4ba0f093c4b5400dbb1776ff5c0416e203b
From: Yannick Tivisse
Date: 2018-09-10 12:23:42

Breaking data model changes scores: del:6 + add:0, change matches:
-    _inherit = 'res.users'
-    has_group_sale_pricelist = fields.Boolean(
-    has_group_pricelist_item = fields.Boolean(
-    has_group_product_pricelist = fields.Boolean(
-    has_group_stock_packaging = fields.Boolean(
-    has_group_product_variant = fields.Boolean(

Total Changes: 29

Revert "[IMP] *: Define groups on res.users models"

This reverts commit 99f497b390e680629dce3a7e2e282ada74ee9884.

================================= pseudo patch: =================================

--- a/addons/product/models/__init__.py
+++ b/addons/product/models/__init__.py
@@ -15,4 +15,3 @@ from . import product_attribute
 from . import product_pricelist
 from . import res_company
 from . import res_partner
-from . import res_users

--- a/addons/product/models/res_users.py
+++ b/None
@@ -1,28 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import fields, models
-
-
-class ResUsers(models.Model):
-    _inherit = 'res.users'
-
-    has_group_sale_pricelist = fields.Boolean(
-        'Sales Pricelists', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        group_xml_id='product.group_sale_pricelist')
-
-    has_group_pricelist_item = fields.Boolean(
-        'Manage Pricelist Items', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        group_xml_id='product.group_pricelist_item')
-
-    has_group_product_pricelist = fields.Boolean(
-        'Pricelists On Product', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        group_xml_id='product.group_product_pricelist')
-
-    has_group_stock_packaging = fields.Boolean(
-        'Manage Product Packaging', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        group_xml_id='product.group_stock_packaging')
-
-    has_group_product_variant = fields.Boolean(
-        'Manage Product Variants', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        group_xml_id='product.group_product_variant')
